### ISSUE TITLE
PMO: No Alt Attribute for Homepage Image in In-Depth Guide

### ISSUE THREAD

**Issue by starlap** [donejs/donejs#655]
**There is no alt attribute for the homepage image.**
![homepage_image_alt](https://cloud.githubusercontent.com/assets/18269875/14694496/c936b67e-071d-11e6-82ea-27a7c9b39f7c.png)

**Comment by justinbmeyer** [comment]
Is this causing a warning?

**Comment by phillipskevin** [comment]
None of the images in the guide have alt attributes and the guide isn't really focused on accessibility, so simply adding alt attributes doesn't really add anything to the guide. If anyone is interested in adding a focus on accessibility to the guide or a new guide, please open a separate issue and we can figure out how best to do that.

**Comment by chasenlehara** [comment]
I think adding `alt` attributes to the images is a very small cost to avoiding errors if someone validates the HTML: https://validator.w3.org/nu/?doc=http%3A%2F%2Fwww.place-my-order.com%2F
<img width="1318" alt="screen shot 2017-02-07 at 9 33 51 am" src="https://cloud.githubusercontent.com/assets/10070176/22703308/8fbc512e-ed18-11e6-8881-fedfde21ec53.png">

[Also, let me know if you want me to open up issues for the other warnings & errors that the validator finds.]

**Comment by phillipskevin** [comment]
There are plenty of other errors that we're not going to be able to fix. These for example:

```
Attribute {($value)} not allowed on element input at this point.
```

Is there a reason someone would validate the HTML? Are there text editors that do this automatically or anything like that?

**Comment by phillipskevin** [comment]
You can see the errors I'm talking about at any of these links:

https://validator.w3.org/nu/?doc=http://www.place-my-order.com/restaurants
https://validator.w3.org/nu/?doc=http://www.place-my-order.com/restaurants/crab-bar/order
https://validator.w3.org/nu/?doc=http://www.place-my-order.com/order-history

**Comment by matthewp** [comment]
W3 validation isn't really relevant any more, most web pages like google.com do not validate correctly. However, I'm with @chasenlehara that adding an alt is very low cost and accessibility is hugely important, so I say we fix this one.

**Comment by justinbmeyer** [comment]
I think we fix it commensurate with its `P4` priority.

**Comment by matthewp** [comment]
Fixed in 4335f313a0450544b62d7a50ac61cf6ff0f8b52e
